src/GoogleAdWordsEventForwarder.js

```diff
@@ -269,6 +323,31 @@
             return true;
         }
 
+        // Creates a new Consent State Payload based on Consent State and Mapping
+        function generateConsentStatePayloadFromMappings(consentState, mappings) {
+            var payload = {};
+    
+            for (var mappingEntry of consentMappings) {
+                if (
+                    consentState[mappingEntry.map] &&
+                    mappingEntry.maptype === 'ConsentPurposes' &&
+                    googleConsentProperties.includes(mappingEntry.value)
+                ) {
+                    payload[mappingEntry.value] = consentState[
+                        mappingEntry.map
+                    ].Consented
+                        ? googleConsentValues.GRANTED
+                        : googleConsentValues.DENIED;
+                }
+            }
+    
+            return payload;
+        }
+
+        function hashConsentPayload(payload) {
```

Comment:
This should be renamed because it's not hashing the consent payload.  instead it's just concatenating the keys/values into a string for future comparison.  Maybe something like `stringifyPayload` or similar?